src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/HttpSamDAO.scala

```diff
@@ -25,6 +26,10 @@ import scala.concurrent.{ExecutionContext, Future}
 class HttpSamDAO( implicit val system: ActorSystem, implicit val executionContext: ExecutionContext )
   extends SamDAO with RestJsonClient {
 
+  override def listWorkspaceResources(implicit userInfo: WithAccessToken): Future[Seq[UserPolicy]] = {
+    authedRequestToObject[Seq[UserPolicy]](Get(samListResources("workspace")))
+  }
```

Comment:
fyi - and no action necessary for this PR - add a `label` if you want to see performance logging on this. See `registerUser` just  below as an example.